docs/installation/reference-installer.mdx

```diff
@@ -111,21 +75,48 @@ operating systems.
     svg="linux" />
 </Install>
 
+## `netdata-installer.sh`
+
+The following installation methods are a wrapper around a single script,
+[`netdata-installer.sh`](https://raw.githubusercontent.com/netdata/netdata/master/netdata-installer.sh), which handles
+the installation process itself:
+
+- [One-line installer (`kickstart.sh`)](/packaging/installer/methods/kickstart.md)
+- [Pre-compiled static binary (`kickstart-static64.sh`)](/packaging/installer/methods/kickstart64.md)
+- [Manual from Git](/packaging/installer/methods/manual.md)
+- [Build from source](/packaging/installer/methods/source.md)
+- [Offline nodes](/packaging/installer/methods/offline.md)
+
+`netdata-installer.sh` is, essentially, an automated version of the [manual from
+Git](/packaging/installer/methods/manual.md) method.
+
+### Parameters
+
+The [`netdata-install.sh` script](https://raw.githubusercontent.com/netdata/netdata/master/netdata-installer.sh) takes
+the following parameters to customize any installation of Netdata.
+
+| Parameter             | Description                                                                                                                                                                         |
+| :-------------------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `--dont-wait`         | Skip prompting for permission to install any required packages. Useful for automated installation, such as with [Ansible](/docs/guides/ansible.md).                                 |
+| `--dont-start-it`     | Prevent the installer from starting Netdata automatically.                                                                                                                          |
+| `--stable-channel`    | Automatically update only on the release of new major versions. See [nightly vs. stable releases](#nightly-vs-stable-releases).                                                                                                                    |
+| `--nightly-channel`   | Automatically update on every new nightly build. See [nightly vs. stable releases](#nightly-vs-stable-releases).                                                                                                                                   |
```

Comment:
I think we should mention at this point of the guide that this is the default behaviour of the Agent. So in reality it's about either using `--stable-channel` flag or no flag at all.

Reply:
Maybe an enhancement for the next "pass" on this front would be to use the individual method pages to explain the default. For example, the `kickstart.sh` page would explain that by running `bash <(curl -Ss https://my-netdata.io/kickstart.sh)`, you get:

- Automatic updates
- Nightly updates
- Anonymous statistics on

Then the page can reference back to this table to help users who want to override those.